packages/market-service/src/coincap/generateMarketCapData.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs'
+
+import { findAll } from '..'
+
+const generateMarketCapData = async () => {
```

Comment:
we shouldn't store and cache market cap data like we do for assets.

asset data is essentially static so we can generate it offline periodically. market data changes constantly so we can't cache it statically.

Reply:
I did this following the coingecko example, can remove if needed

Reply:
ya, please go ahead and delete this file, the generated data, the generate script, and the scripts in package.json - i don't know why i wrote this initially :\